### PR TITLE
feature(Implement Supervision Depot)

### LINKED ISSUE #6877: Problems of the management and supervision of deposits
We have noticed that there may be a user who can only monitor repositories he has access rights to, but cannot monitor repositories he does not have access rights to. access, especially when the option "Limit stock by user's deposit permissions" is activated

The solution is to add a new option, which will allow a user to be able to supervise a repository to which he does not have access rights

### PR DESCRIPTION
**Pull request by lomamech**
- Add new option for supervision of depots
- Add option to select a depot and dont select this child
- Implement integration test

closes #6877

### CODE REVIEW

**Comment by mbayopanda** [comment]
Chris peux-tu mettre à jour ta branch ?

File: server/controllers/admin/users/depotsSupervision.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/**
+ * @overview Users/Permissions
```
Comment (mbayopanda) [inline server/controllers/admin/users/depotsSupervision.js:2]: Does this `@overview` is correct?

File: server/controllers/admin/users/depotsSupervision.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ *
+ * @description
+ * Provides helper functionality to the /users routes.  This file groups all
+ * depots-related functionality in the same place.
```
Comment (mbayopanda) [inline server/controllers/admin/users/depotsSupervision.js:6]: Could you be more explicit about this function?

**Comment by lomamech** [comment]
> Chris peux-tu mettre à jour ta branch ?

Ok

**Comment by lomamech** [comment]
@mbayopanda , I just solved the conflict with the branch master

**Review by mbayopanda — APPROVED** [review]
LGTM 👍🏽 

bors r+

**Pull request merged**